docs/runbooks/Container-README.md

```diff
@@ -1,373 +1,71 @@
-      BACKEND_PORT: ${BACKEND_PORT}
-      BACKEND_INTERFACE: ${BACKEND_INTERFACE}
-      BACKEND_ALLOWED_ORIGINS: ${BACKEND_ALLOWED_ORIGINS}
-      BACKEND_LOG_FILTER_LEVEL: ${BACKEND_LOG_FILTER_LEVEL}
-      TIPTAP_URL: ${TIPTAP_URL}
-      TIPTAP_AUTH_KEY: ${TIPTAP_AUTH_KEY}
-      TIPTAP_JWT_SIGNING_KEY: ${TIPTAP_JWT_SIGNING_KEY}
-    ports:
-      - "${BACKEND_PORT}:${BACKEND_PORT}"
-    depends_on:
-      - postgres
-    networks:
-      - backend_network
-    command: ["sh", "-c", "sleep 5 && /usr/local/bin/refactor_platform_rs"]
+   git clone <repository-url> && cd <repository-directory>
+  ```
   
-  nextjs-app:
-    build:
-      context: https://github.com/refactor-group/refactor-platform-fe.git#main
-      dockerfile: Dockerfile
-      target: runner
-      args:
-        NEXT_PUBLIC_BACKEND_SERVICE_PROTOCOL: ${BACKEND_SERVICE_PROTOCOL}
-        NEXT_PUBLIC_BACKEND_SERVICE_PORT: ${BACKEND_PORT}
-        NEXT_PUBLIC_BACKEND_SERVICE_HOST: ${BACKEND_SERVICE_HOST}
-        NEXT_PUBLIC_BACKEND_API_VERSION: ${BACKEND_API_VERSION}
-        FRONTEND_SERVICE_PORT: ${FRONTEND_SERVICE_PORT}
-        FRONTEND_SERVICE_INTERFACE: ${FRONTEND_SERVICE_INTERFACE}
-    environment:
-      NEXT_PUBLIC_BACKEND_SERVICE_PROTOCOL: ${BACKEND_SERVICE_PROTOCOL}
-      NEXT_PUBLIC_BACKEND_SERVICE_PORT: ${BACKEND_PORT}
-      NEXT_PUBLIC_BACKEND_SERVICE_HOST: ${BACKEND_SERVICE_HOST}
-      NEXT_PUBLIC_BACKEND_API_VERSION: ${BACKEND_API_VERSION}
-    ports:
-      - "${FRONTEND_SERVICE_PORT}:${FRONTEND_SERVICE_PORT}"
-    depends_on:
-      - rust-app
-
-networks:
-  backend_network:
-    driver: bridge
-
-volumes:
-  postgres_data
-```
-
----
-
-## Building and Running the Application
-
-### **1. Build the Rust Backend Image**
-
-```bash
-docker buildx build --platform linux/amd64,linux/arm64 -t rust-backend .
-```
-
-This builds the image for both `amd64` and `arm64` architectures. Use the `--platform` flag to build for a specific architecture.
-
-### **2. Build and Run with Docker Compose**
-
-#### For Local PostgreSQL:
-
-```bash
-docker-compose --env-file .env.local up --build
-```
-
-#### For Remote PostgreSQL:
-
-```bash
-docker-compose --env-file .env.remote-db up --build
-```
-
-The web API will be accessible at `http://localhost:<SERVICE_PORT>`
-
----
-
-## Database Utilities
-
-### **Rebuild the Database**
-
-```bash
-docker-compose run rust-app rebuild-db
-```
-
-### **Seed the Database**
-
-```bash
-docker-compose run rust-app seed-db
-```
-
-### **Convert DBML to SQL**
+## Copy the example .env file and adjust values as needed
 
-If you have a DBML file (`schema.dbml`), convert it to SQL:
-
-```bash
-docker-compose run -v $(pwd)/sql:/app/sql -v $(pwd)/schema.dbml:/app/schema.dbml rust-app dbml2sql
-```
-
-```bash
-docker-compose run -v $(pwd)/sql:/app/sql -v $(pwd)/schema.dbml:/app/schema.dbml rust-app dbml2sql
-```
-
----
-
-## Managing Containers
+  ```bash
+    cp .env.example .env
```

Comment:
I don't see a .env.example file here in the docs directory, can you please add one that includes all of the necessary environment variables set to real values that work?